### ISSUE TITLE
Intent gets switched in the middle of a story

### ISSUE THREAD

**Issue by mikob** [wit-ai/wit#303]
How do I prevent wit from changing the story/intent, when it's in the middle of a story? For some reason when a one-word answer which is mapped as a subject intent is given, wit changes to a  different story associated with a different action. Anyway to prevent this?
![screenshot from 2016-08-22 14-51-53](https://cloud.githubusercontent.com/assets/492864/17844879/c7b6518a-6878-11e6-9897-851ef25c887b.png)
![screenshot from 2016-08-22 14-52-46](https://cloud.githubusercontent.com/assets/492864/17844880/c7bd2b4a-6878-11e6-95fe-f7a081af0577.png)

**Comment by Bamsoo** [comment]
Hi,
I had the same problem, because for some reason I imagined the original intent (trait detected in the story's first input) would carry along the whole story. This is not the case. 
I found no other solutions that inserting a context key at the beginning of every story, so that decision for the next input will be inside the story's action. This can be reinforced using "only if.." in the actions tab. This context concept is a bit misleading in the docs, imho.
